.github/.codecov.yaml

```diff
@@ -0,0 +1,3 @@
+codecov.notify.after_n_builds: 2
```

Comment:
shouldn't it be 4 (3 builds for python, 1 for java?)

Reply:
Hmm... maybe? Not a lot of docs. I was thinking it was only builds that created jacoco coverage reports. But I guess that is 4 too, isn't it. For some reason I thought we saw all the python reports in a single operation.